test-infra/aws-vpn/locals.tf

```diff
@@ -0,0 +1,51 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+locals {
+  global_tags = {
+    "environment" = "josec-vpn-test"
```

Comment:
Perhaps change this to something that doesn't have your name in it 😬

Reply:
LOL i meant to externalize this to a variable, thank you for catching